geodiff/src/geodiff.cpp

```diff
@@ -117,6 +117,31 @@ int GEODIFF_CX_setMaximumLoggerLevel( GEODIFF_ContextH contextHandle,
   return GEODIFF_SUCCESS;
 }
 
+int GEODIFF_CX_setTablesToSkip( GEODIFF_ContextH contextHandle, int tablesCount, const char **tablesToSkip )
+{
+  Context *context = static_cast<Context *>( contextHandle );
+  if ( !context )
+  {
+    return GEODIFF_ERROR;
+  }
+
+  if ( !tablesToSkip )
```

Comment:
it would be probably fair to allow `tablesToSkip` to be null in case `tablesCount` is zero.